docs/encrypted-innodb-tablespace-backups.md

```diff
@@ -22,7 +20,7 @@ These components are stored in the `plugin` directory.
 
```

Comment:
We should explicitly say that plugins are no longer supported.

Reply:
Added